src/Features/Core/Portable/ExternalAccess/VSTypeScript/VSTypeScriptNavigateToSearchService.cs

```diff
@@ -35,30 +35,34 @@ internal class VSTypeScriptNavigateToSearchService : INavigateToSearchService
 
         public bool CanFilter => _searchService?.CanFilter ?? false;
 
-        public async Task SearchDocumentAsync(
+        public async Task<NavigateToSearchLocation> SearchDocumentAsync(
             Document document, string searchPattern, IImmutableSet<string> kinds,
             Func<INavigateToSearchResult, Task> onResultFound,
             bool isFullyLoaded, CancellationToken cancellationToken)
         {
-            if (_searchService == null)
-                return;
+            if (_searchService != null)
+            {
+                var results = await _searchService.SearchDocumentAsync(document, searchPattern, kinds, cancellationToken).ConfigureAwait(false);
+                foreach (var result in results)
+                    await onResultFound(Convert(result)).ConfigureAwait(false);
+            }
 
-            var results = await _searchService.SearchDocumentAsync(document, searchPattern, kinds, cancellationToken).ConfigureAwait(false);
-            foreach (var result in results)
-                await onResultFound(Convert(result)).ConfigureAwait(false);
+            return NavigateToSearchLocation.Latest;
         }
```

Comment:
our bridge methods to F#/TS always return 'Latest' as that's always been the semantics they expose.  they can always come and decide to do things differnetly here if they want (for example supporting returning cached results with the same semantics as us if they want.)